lib/src/builders/flags_builder/assets_to_cache_builder.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:build/build.dart';
+
+class AssetToCacheBuilder implements Builder {
+  AssetToCacheBuilder();
+
+  @override
+  Map<String, List<String>> get buildExtensions => {
+        '.svg': const ['.part.txt']
+      };
+
+  @override
+  FutureOr<void> build(BuildStep buildStep) async {
+    final inputId = buildStep.inputId;
+
+    print("Generating ${inputId.path}.part.txt");
+
+    final baseName = inputId.pathSegments.last;
+    final name = baseName.substring(0, baseName.length - 4); // cut .svg
```

Comment:
nit: there is a dart method for this, I believe it's something like baseNameWithoutExtension or something like that